src/test/java/Homework17.java

```diff
@@ -0,0 +1,21 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest{
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+    String newSongNotification = "Added 1 song into";
+    urlAccess();
+    getEmail("andalia.nia@gmail.com");
+    getPassword("te$t$tudent");
+    clickSubmit();
+    searchSong("Pluto");
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/222109579-75821639-4589-479b-a137-f5979224974b.png)

Hint 

![image](https://user-images.githubusercontent.com/104761482/222110059-ff6a5b2a-8f63-49a0-83b7-f7d15cfdf2f7.png)
```
    public void searchSong(String songName) throws InterruptedException{
        WebElement searchSong = driver.findElement(By.cssSelector("[type='search']"));
        searchSong.sendKeys(songName);
        Thread.sleep(2000);
    }
```

Reply:
ok I just submitted an updated file, I'm a bit lost with this one.